package/kedro_viz/integrations/kedro/data_loader.py

```diff
@@ -85,10 +87,15 @@ def load_data(
         A tuple containing the data catalog and the pipeline dictionary
         and the session store.
     """
-    from kedro.framework.project import pipelines
-    from kedro.framework.startup import bootstrap_project
+    from kedro.framework.project import configure_project, pipelines
```

Comment:
I wonder why these imports are inside the function? Performance, circular imports, something else? Would be good to add a comment

Reply:
Good catch. They no longer need to be inside function -- earlier when we were supporting kedro 17 and kedro 16, they way to import was different for each version of Kedro hence at that point, they were inside the function but that is no longer required.

Reply:
Thanks @astrojuanlu for raising this. Even I used to wonder if there is a performance benefit. I found out that there could be, if our imports are heavy and are used only in a single function, then it makes sense to have it inside a function both for readability and performance. In our case, it is not needed. I am moving it to the top import section.